lib/map_elevation.dart

```diff
@@ -18,35 +18,40 @@ class Elevation extends StatefulWidget {
 
   /// Elevation gradient colors
   /// See [ElevationGradientColors] for more details
-  final ElevationGradientColors elevationGradientColors;
+  final ElevationGradientColors? elevationGradientColors;
 
   /// [WidgetBuilder] like Function to add child over the graph
-  final Function(BuildContext context, Size size) child;
+  final Function(BuildContext context, Size size)? child;
 
   Elevation(this.points,
-      {this.color, this.elevationGradientColors, this.child});
+      {required this.color, this.elevationGradientColors, this.child});
```

Comment:
Worth adding this breaking change in CHANGELOG.md

Reply:
Hmm there's no point making it required, so I'll default it to transparent ;)

Reply:
Well it seems the painter isn't painting with transparent color. I'll mark it as a BC.